google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
@tst-dhudlow fundamentally, moving the DOM is _absolutely_ the right thing to do. I also agree that preserving event handlers is vitally important. 

I don't really understand what's happening during the ripple/ink effect, but it's not (as I had thought) that the infowindow content is destroyed - it's still there, but the display of the content in the browser is gone (see pics).

before clicking paper-icon-button +
![p294-ink-a](https://cloud.githubusercontent.com/assets/1044471/16096808/ebc54334-3307-11e6-902c-4dc040ab5ac4.png)
after clicking paper-icon-button
![p294-ink-b](https://cloud.githubusercontent.com/assets/1044471/16096813/f1a51bda-3307-11e6-9cba-45b54be896de.png)
Demo Code 
[ink-demo-pull-294.zip](https://github.com/GoogleWebComponents/google-map/files/317192/ink-demo-pull-294.zip)